### PR TITLE
Added camelize function to CommonServerPython

### PR DESCRIPTION
**Pull request by BenJoParadise**

### CODE REVIEW

**Comment by BenJoParadise** [comment]
Tested the camelize function after the commit that states that it's untested.

**Comment by ronykoz** [comment]
@anara123 you are ok with adding such a function, right?
It will make our lives easier when converting human readable to context format

File: Scripts/script-CommonServerPython.yml [hunk 1]
```diff
@@ -1234,4 +1256,4 @@ dependson: {}
 timeout: 0s
 tests:
   - TestPYCommonServer
-releaseNotes: "Add djb2 hash function"
```
Comment (ronykoz) [inline Scripts/script-CommonServerPython.yml]: Not sure this should be removed

Comment (BenJoParadise) [reply Scripts/script-CommonServerPython.yml]: It's not my release note, i think this goes to Alex after every merge (it's there from the last merge of this file to Master).

Not sure though, how can we know for sure?

Comment (ronykoz) [reply Scripts/script-CommonServerPython.yml]: These releaseNotes are staying there until Itay removes them after the release, thus these release notes are there for the next release, you can check with him to make sure of that

Comment (Itay4) [reply Scripts/script-CommonServerPython.yml]: @ronykoz is right. In this cases, just add numbering and append your release notes.

**Comment by BenJoParadise** [comment]
@anara123 i followed @ronykoz steps, can you please force merge?

**Pull request merged**